doc-include/migrate-mac.dtx

```diff
@@ -358,14 +358,14 @@
 %   \item \cs{unvxh} has been replaced with \cs{Xunvxh} and \cs{unvxhX} with two mandatory arguments.
 % \end{itemize}
 % \subsection{Migration to \reledmac~2.1.0}
-% \Reledmac~2.1.0 fix some bug when using \cs{Xbhooknote} and \cs{bhooknoteX} not in order to execute code at the beginning of each notes, but to insert content of at the beginning of each notes.
+% \Reledmac~2.1.0 fix some bugs when using \cs{Xbhooknote} and \cs{bhooknoteX}, in order to insert content of at the beginning of each note. This is not intended to execute code at the beginning of each note.
```

Comment:
hum, I am sure you say the same thing.
I : people has used bhooknoteX to insert content. It is bad. It should be only to insert code.
You : people use bhooknoteX to insert content. It does not matter. bhooknoteX should not use to insert code.

So it is exactly oppositie